Tests/MapboxSearchUITests/Integration/CategorySuggestionsIntegrationTestCase.swift

```diff
@@ -1,16 +1,18 @@
 import XCTest
 
-class CategorySuggestionsIntegrationTestCase: MockSBSServerUITestCase {
+// Rename to UITestCase
```

Comment:
Most of the UI test cases have "Integration" in the name rather than "UI" and this conflicts with the actual integration tests which _also_ have "integration" in their names. 

This rename operation and other clean-up changes (split MockResponse enum instances into separate files, rename MockResponse cases) can occur in a separate PR to reduce noise.